app/src/main/java/ca/bc/gov/bchealth/ui/comment/CommentsSummaryUI.kt

```diff
@@ -0,0 +1,113 @@
+package ca.bc.gov.bchealth.ui.comment
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.pluralStringResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import ca.bc.gov.bchealth.R
+import ca.bc.gov.bchealth.compose.BasePreview
+import ca.bc.gov.bchealth.compose.MyHealthTheme
+import ca.bc.gov.bchealth.compose.MyHealthTypography
+import ca.bc.gov.bchealth.compose.blue
+import ca.bc.gov.bchealth.compose.grey
+import ca.bc.gov.bchealth.compose.greyBg
+import ca.bc.gov.common.utils.toDateTimeString
+import java.time.Instant
+
+@Composable
+fun CommentsSummaryUI(
```

Comment:
@bsavoini-fw , I am thinking, we should not do tight coupling with viewModel for the ui components that we are expecting to be reused in other screen.
You should remove view model constructor and only expose the constructor with commentSummary & click lambda.
What do you think?

Reply:
Yeah, I think it makes sense for reusable components. I'll update that